DFE-Digital/Dfe.Academies.External
# 108413 remove a contributor

See ticket:-
https://dfe-gov-uk.visualstudio.com.mcas.ms/Academies-and-Free-Schools-SIP/_workitems/edit/108413?McasTsid=26110

## Type of change

## Expected behaviour
User can remove a contributor from invite contributors page

## Steps to reproduce issue (if relevant)
n/a

## Steps to test this PR
1. from your applications go into an application
2. go to invite contributors link at bottom of page
3. on invite contributors page there's a list of contributors at the bottom, alongside each one is remove link
4. click remove link, user show confirm page
5. click remove removes form API / DB
6. user sent back to invite contributors page, contributor will no longer exist

## Prerequisites
n/a
